docs/apps/lnw/es2k/es2k-lnw-overlay-vms.md

```diff
@@ -205,12 +230,25 @@ Example:
 - Corresponding port representor VSI value 9
 - If a VSI is used as an action, add an offset of 16 to the VSI value
 
-```bash
-# Create a source port for an overlay VF (VSI-27). Source port action can be any value.
-# For simplicity add 16 to VSI ID.
- p4rt-ctl add-entry br0 linux_networking_control.tx_source_port_v4 \
-     "vmeta.common.vsi=27,zero_padding=0,action=linux_networking_control.set_source_port(43)"
+- For Linux Networking V2
+
+  ```bash
+     # Create a source port for an overlay VF (VSI-27). Source port value should be VSI ID + 16.
+      p4rt-ctl add-entry br0 linux_networking_control.tx_source_port_v4 \
+      "vmeta.common.vsi=27,zero_padding=0,action=linux_networking_control.set_source_port(43)"
+   ```
 
+- For Linux Networking V3
+
+  ```bash
+     # Create a source port for an overlay VF (VSI-27). Source port value should be VSI ID + 16.
+      p4rt-ctl add-entry br0 linux_networking_control.tx_source_port_v4 \
+      "vmeta.common.vsi=27/2047,action=linux_networking_control.set_source_port(43)"
```

Comment:
What is the `/2047` here after the VSI id?

Reply:
it's the mask, since the table is now ternary in v3 instead of SEM